include/swift/Option/Options.td

```diff
@@ -1286,6 +1286,10 @@ def disable_clang_target : Flag<["-"], "disable-clang-target">,
   Flags<[NewDriverOnlyOption]>,
   HelpText<"Disable a separately specified target triple for Clang instance to use">;
 
+def explain_module_dependency : Separate<["-"], "explain-module-dependency">,
```

Comment:
Is this option tested?

Reply:
> Is this option tested?

Yeah, it will be, here:
https://github.com/apple/swift-driver/pull/1363

Reply:
The option is `NewDriverOnly` so only the definition lives here, the driver repo will actually exercise and test it.